src/node_os.cc

```diff
@@ -351,7 +356,8 @@ static void GetUserInfo(const FunctionCallbackInfo<Value>& args) {
   const int err = uv_os_get_passwd(&pwd);
 
   if (err) {
-    return env->ThrowUVException(err, "uv_os_get_passwd");
+    env->CollectUVExceptionInfo(args[1], err, "uv_os_get_passwd");
```

Comment:
For clarity, maybe use `args[args.Length() - 1]`? Also we should probably `CHECK_EQ(args.Length(), n)` or `CHECK_GE(args.Length(), n)` (not sure if these are used in userland)